www/css/main.diary.css

```diff
@@ -136,12 +142,24 @@ a.item-content {
   color: #333;
   font-size: 0.8em;
 } */
+
+
 .btn-input-confirm {
-  line-height: 30px;
+  width: 100%;
+  height: 30px;
+  margin: 2px;
+  box-sizing: border-box;
+  border-radius: 10px;
+  font-family: 'Lato';
```

Comment:
I would be more than willing to remove this font-family. Sure, we don't "need" it, but this font-family is pulled from the savoir-faire linux branch, which the fellow interns found more appealing than the current design.

Reply:
Here is what the default font looks like on the Diary screen
![Simulator Screen Shot - iPhone 11 - 2022-08-31 at 11 48 23](https://user-images.githubusercontent.com/61334340/187745607-a01c973f-a205-4f9a-a0b4-37d7e2aaea70.png)


Here is what the custom font family Lato looks like on the Diary screen
![Simulator Screen Shot - iPhone 11 - 2022-08-30 at 09 47 43](https://user-images.githubusercontent.com/61334340/187745641-79688018-568a-4af1-be09-da66345ad381.png)

I actually don't think the custom font makes much of a difference. Also, since we are not using the Lato font on the dashboard, the Lato font appearing when you switch to the diary screen is a little off-putting, making it feel weird. I believe if we change any font, we should change them all. The Lato font would look good across the entire app, but as far as UX/UI users I don't believe will be impacted much whichever way we go. I will make an issue to remove the Lato font for now.

**_Confirmed this change as complete_**

Reply:
Without the lato font, is the font on the diary and the dashboard the same? It looks to me like the dashboard is sans-serif and at least the filters at the top ("To Label", "All Unlabeled" etc) use a serif font.

Sorry to nitpick like this but this is the kind of issue that people notice and get all picky about.

Reply:
Here is an updated screenshot with the current changes. The fonts all match and it actually looks quite good:

![Simulator Screen Shot - iPhone 14 Pro Max - 2022-09-21 at 10 48 42](https://user-images.githubusercontent.com/61334340/191564310-a7c483ed-d228-41d8-8d4a-e5b82dd89a22.png)

Reply:
_**Confirmed this change as complete**_